internal/policy/commit/check_jira.go

```diff
@@ -0,0 +1,54 @@
+package commit
+
+import (
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/talos-systems/conform/internal/policy"
+)
+
+type JiraCheck struct {
+	errors []error
+}
+
+func (j *JiraCheck) Name() string {
+	return "Jira issues"
+}
+
+func (j *JiraCheck) Message() string {
+	if len(j.errors) != 0 {
+		return j.errors[0].Error()
+	}
+
+	return "Jira issues are invalid"
+}
+
+func (j *JiraCheck) Errors() []error {
+	return j.errors
+}
+
+func (c Commit) ValidateJiraCheck() policy.Check {
+	check := &JiraCheck{}
+	compile := regexp.MustCompile("^(\\w+)( \\w+)?: \\[(\\w+)-(\\d+)\\] .*")
+	if compile.MatchString(c.msg) {
+		submatch := compile.FindStringSubmatch(c.msg)
+		jiraProject := submatch[3]
+		// jiraIssue := submatch[4]
```

Comment:
We can probably get rid of this if we aren't using it?

Reply:
Sure